packages/core/src/hooks/useRawCalls.ts

```diff
@@ -55,5 +56,19 @@ export function useRawCall(call: RawCall | Falsy) {
 
 function extractCallResult(chains: MultiChainState, call: RawCall): RawCallResult {
   const chainId = call.chainId
-  return chainId !== undefined ? chains[chainId]?.value?.state?.[call.address.toLowerCase()]?.[call.data] : undefined
+  if (chainId !== undefined) {
+    const rawCallResult = chains[chainId]?.value?.state?.[call.address.toLowerCase()]?.[call.data]
+    if (rawCallResult) {
+      return rawCallResult
+    }
+    const error = chains[chainId]?.value?.error as any
+    const errorMessage = error?.error?.message ?? error?.message
```

Comment:
I hope we don't need to add more, like here:

https://github.com/TrueFiEng/useDApp/blob/c074e166e1f80b5567d8c002ec06dcb16abcfe8f/packages/core/src/hooks/usePromiseTransaction.ts#L84